src/AutoRest.CSharp/Common/Generation/Writers/CodeWriterExtensions.cs

```diff
@@ -274,6 +281,11 @@ private static IDisposable WriteMethodDeclarationNoScope(this CodeWriter writer,
 
         public static CodeWriter WriteMethodDocumentation(this CodeWriter writer, MethodSignatureBase methodBase)
         {
+            if (methodBase.IsRawSummaryText)
+            {
+                return writer.WriteRawXmlDocumentation(methodBase.Description);
+            }
```

Comment:
OK this is how it is used.
But I do not really like we put a flag here.
What would it happen if we just use the same method as we write ordinary descriptions?

Reply:
The xml documentation from base contract contains both summary description and parameter list.
Another option is to extract summary description only and use it with the original `WriteXmlDocumentationSummary`.

Reply:
I am OK with this right now, but I believe later we need a clean up on this because this flag does not make much sense.